tests/unit/test_middleware/test_session/test_integration.py

```diff
@@ -16,15 +15,17 @@
 from litestar.testing import create_test_client
 
 
-class User(BaseModel):
+@dataclass
+class User:
     id: UUID
     name: str
-    email: EmailStr
```

Comment:
No need for Pydantic specific tests here; We test handling of those custom types in the Pydantic plugin tests